### PR TITLE
FaceID Control Loop [GG-76]

### PR DESCRIPTION
**Pull request by curlyjay**
Implements control loop to manage PIR motion detection, face detection, and face recognition
Includes .gitignore updates

### CODE REVIEW

File: FaceID/faceID.py [hunk 1]
```diff
@@ -205,18 +204,18 @@ def removeEmployee():
...
         for person in photoRegister:
             extractEmpID = int(person[1].split("_")[0])
-        
+
             if delEmpID == extractEmpID:
                 print("Removing Employee ID: "+str(extractEmpID)+" Pictures . . .")
-                for pic in person: 
-                    response = req.delete(url + "employees/" + str(extractEmpID) + "/")
+                for pic in person:
+                    response = req.delete(url + "employees/" + str(extractEmpID) + "/", headers=headers)
```
Comment (orndorffc) [inline FaceID/faceID.py]: fix spacing around =

File: FaceID/faceID.py [hunk 2]
```diff
@@ -323,9 +327,14 @@ def searchFace():
...
-                if (avgAccuracy > 25):
+                if (avgAccuracy > accThreshold):
 
-                    response = req.get(url + "employees/" + str(EmpID) + "/")
+                    if lock == True: # things to run once ! 
+                        print("UNLOCKED")
+                        lock = False
+                        
+
+                    response = req.get(url + "employees/" + str(EmpID) + "/", headers=headers)
```
Comment (orndorffc) [inline FaceID/faceID.py]: Fix spacing

File: FaceID/faceID.py [hunk 3]
```diff
@@ -361,16 +370,24 @@ def searchFace():
                                     imageFile.name = photoName + ".jpg"
                                     files = {"file" : imageFile}
                                     if photoName != "none":
-                                        response = req.post(url + "pictures/" + str(EmpID) + "/", files=files) # add newly captured photo
-                                    
+                                        response = req.post(url + "pictures/" + str(EmpID) + "/", files=files, headers=headers) # add newly captured photo
```
Comment (orndorffc) [inline FaceID/faceID.py]: fix spacing

**Review by orndorffc — APPROVED** [review]
Works as planned.

File: FaceID/faceID.py [hunk 4]
```diff
@@ -285,35 +282,42 @@ def searchFace():
...
+                timeStamp = time.time()
+
                 EmpID, accuracy = recognizer.predict(gray[y:y+h,x:x+w]) # recognizes face
-                
+
                 accuracy = round(100 - accuracy)
-                    
-                if (0 < accuracy <= 100):
+
+                if (accuracy <= 100) and (accuracy > 0):
```
Comment (mraynor) [inline FaceID/faceID.py]: This can be simplified to: `if (0 < accuracy <= 100):`

Comment (curlyjay) [reply FaceID/faceID.py]: Fixed!

**Pull request merged**